drivers/network/tcpip/ip/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ list(APPEND SOURCE
 add_library(ip OBJECT ${SOURCE} ${ip_asm})
 
 target_link_libraries(ip lwipcore)
+add_importlibs(lwipcore ntoskrnl)
```

Comment:
This line shouldn't be needed for the lwipcore library. It should instead be added for the final build targets like the actual drivers. (For example the tcpip driver has it in its cmakelists.)

Reply:
Without it, I get undefined references to strncmp, atoi, isxdigit, islower and isspace.